src/NUnitFramework/framework/Constraints/Operators/ExactCountOperator.cs

```diff
@@ -39,17 +39,24 @@ public class ExactCountOperator : CollectionOperator
         /// <param name="expectedCount">The expected count</param>
         public ExactCountOperator(int expectedCount)
         {
+            // Collection Operators stack on everything
```

Comment:
This is copied from the former base constructor in CollectionOperator. I don't particularly like this.